### ISSUE TITLE
Worms World Party Remastered - Black Screen

### ISSUE THREAD

**Issue by 7oxicshadow** [doitsujin/dxvk#4569]
When trying to play WWPR you appear to get a single frame of the Team 17 logo and then the screen just goes black with the audio playing in the background.

Researching this issue, it appears that this is a problem with Windows 10 as well but there is a workaround to get it playable again.

The solution for Windows 10 is to press the right mouse button on the w2.exe and select "Disable Fullscreen Optimizations".

As we cannot do the same thing in Linux (wine) is there an equivalent option within DXVK that might do the same thing? On Proton DB some users have reported success using the steam overlay and some alt-tab switching but for me it normally seems to crash the game.

I have attached a DXVK log file in case it shows any possible causes.

[w2_d3d9.log](https://github.com/user-attachments/files/18281553/w2_d3d9.log)

**Comment by K0bin** [comment]
Test master. This is most likely already fixed.

https://github.com/doitsujin/dxvk/actions/runs/12435734651

**Comment by Riesi** [comment]
This is a wine bug.
The game renders its input as a separate windows on top of the graphics window.

https://bugs.winehq.org/show_bug.cgi?id=14543

**Comment by K0bin** [comment]
> The solution for Windows 10 is to press the right mouse button on the w2.exe and select "Disable Fullscreen Optimizations".

FWIW DXVK 2.5.2 does that by default.

**Comment by 7oxicshadow** [comment]
@Riesi thanks for the wine report. I think that is for the original worms world party. That used ddraw for graphics. This is for the remaster which uses d3d9. I suppose it is possible that they are the same bug but would be surprised if it was

**Comment by 7oxicshadow** [comment]
> > The solution for Windows 10 is to press the right mouse button on the w2.exe and select "Disable Fullscreen Optimizations".
> 
> FWIW DXVK 2.5.2 does that by default.

I have tried latest master and it still happens for me sadly

**Comment by K0bin** [comment]
Yeah sorry, I didn't make that clear. This only affects Windows. Linux has no concept of exclusive fullscreen.
